### PR TITLE
Button is now div

### PR DESCRIPTION
**Pull request by CHedgren**
This should fix the example in Firefox.
closes: #10

### CODE REVIEW

**Comment by ericgio** [comment]
@CHedgren: this looks pretty good, thanks. I have a couple comments on it, would you prefer to make the changes yourself or would you rather I patch and do it myself?

File: src/Token.react.js [hunk 1]
```diff
@@ -51,11 +51,13 @@ const Token = React.createClass({
         onFocus={this._handleSelect}
         onKeyDown={this._handleKeyDown}
         tabIndex={0}>
-        {this.props.children}
-        <span className="close-button" onClick={this._handleRemove}>
+        <span>
```
Comment (ericgio) [inline src/Token.react.js]: I don't think the `span` here is necessary, let's omit it.

File: src/Token.react.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ const Token = React.createClass({
 
   _renderRemoveableToken() {
     return (
-      <button
+        <div
```
Comment (ericgio) [inline src/Token.react.js]: Did you consider using an `<a>` instead? Given that the removable token is meant to be interactive, an `<a>` might reflect that a bit better.

**Comment by CHedgren** [comment]
As for the span around the props, that is what we are actually rendering from react, i added it to be explicit about it. I suggest you remove it once we use 0.15. 
See this pr for more info: https://github.com/facebook/react/pull/5753/commits/203850013a42f1cde6fba1dea3c26c4b40e6cc90

I didn't consider using an anchor tag instead, an anchor tag defines a hyperlink to another page or some location on the current page and that is not the kind of affordance offered by a thing that looks like a button. 

Is it only the X-mark that is supposed to be interactive? Because then it would be cleaner, but equivalent to do it as a 
`<div>
   <span> text</span><button> X </button
</div>`

**Comment by ericgio** [comment]
Both the token itself and the "x" are interactive. Making the "x" a button still seems like a good idea, though.

I don't see any reason to be explicit about rendering `span`s, especially since that's going away. Could you please remove those and squash everything into a single commit and I'll merge this? Thanks.

**Comment by ericgio** [comment]
Thanks for reporting the issue and submitting a fix! I ended up reverting back to a `span` for the `close-button` because using a `button` added a separate focus state, which I didn't want.

**Pull request closed without merging**